gdx-ai/src/com/badlogic/gdx/ai/btree/branch/Parallel.java

```diff
@@ -138,9 +148,96 @@ public void reset () {
 	protected Task<E> copyTo (Task<E> task) {
 		Parallel<E> parallel = (Parallel<E>)task;
 		parallel.policy = policy; // no need to clone since it is immutable
-
+		parallel.strategy = strategy; // no need to clone since it is immutable
 		return super.copyTo(task);
 	}
+	
+	public void resetAllChildren() {
+		for (int i = 0, n = getChildCount(); i < n; i++) {
+			Task<E> child = getChild(i);
+			child.reset();
+		}
+	}
+	
+	/** The enumeration of the child execution strategies supported by the {@link Parallel} task */
+	public enum Strategy {
```

Comment:
How about you rename this enum to `Orchestrator`, `Executor` or something that suggests children coordination?
I mean, the difference between `Strategy` and `Policy` is really not intuitive just by looking at their names.